webpack-modules/README.md

```diff
@@ -0,0 +1,31 @@
+# WebExtension Webpack Example
+A minimal example of how to use npm modules from within a WebExtension.
```

Comment:
Some links would be good here, and a reference to webpack:

"A minimal example of how to use [webpack](https://webpack.github.io/) to package [npm](https://www.npmjs.com/) modules so they can be used in a WebExtension."